legend-engine-pure-code-compiled-core/src/main/resources/core/pure/protocol/vX_X_X/invocations/execution.pure

```diff
@@ -152,6 +157,7 @@ function meta::protocols::pure::vX_X_X::invocation::execution::execute::buildExe
                                           _type = 'pure',
                                           baseVersion = '-1', //meta::vcs::svn::metamodel::getCurrentSvnRevisionNumber('/system')->toString(),
                                           version = 'none',
+                                          overrideUrl = $pureMetadataServer,
```

Comment:
Also, can we have a simple unit test which asserts that override url is set

Reply:
Added